src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/base/ImageComponent.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from 'react';
+import { useSelector } from 'react-redux';
+import { Grid, makeStyles } from '@material-ui/core';
+import { ITextResourceBindings } from 'src/features/form/layout';
+import { HelpTextContainer } from 'src/features/form/components/HelpTextContainer';
+import { IRuntimeState } from '../../types';
+
+export interface IImageProps {
+  id: string;
+  imageSpecs: IImagesSpecifics;
+  textResourceBindings: ITextResourceBindings;
+  commonPhrases: any;
+  getTextResource: (key: string) => string;
+  getTextResourceAsString: (key: string) => string;
+}
+
+export interface IImagesSpecifics {
+  imageSrc: IImageSrc;
+  imgWidth: string;
+  align: string;
+}
+
+export interface IImageSrc {
+  nb: string;
+  nn: string;
+  en: string;
+}
+
+const useStyles = makeStyles({
+  spacing: {
+    letterSpacing: '0.3px',
+  },
+});
+
+export function ImageComponent(props: IImageProps) {
+  const classes = useStyles();
+  const language: string =
+    useSelector((state: IRuntimeState) => state.profile.profile.profileSettingPreference.language);
+  const width = props.imageSpecs.imgWidth || '100%';
+  const altText = props.getTextResourceAsString(props.textResourceBindings.altTextImg) || 'image';
+
+  let imgSrc;
+  if (language === 'en' && props.imageSpecs.imageSrc.en) {
+    imgSrc = props.imageSpecs.imageSrc.en;
+  } else if (language === 'nn' && props.imageSpecs.imageSrc.nn) {
+    imgSrc = props.imageSpecs.imageSrc.nn;
+  } else {
+    imgSrc = props.imageSpecs.imageSrc.nb;
+  }
+
+  let align;
+  if (props.imageSpecs.align === 'right') {
+    align = 'flex-end';
```

Comment:
I see no problem with that as long as it is documented 👍

Reply:
Then I'll remove the abstraction 😊

Reply:
It.. does make the linting angry though... Which is weird because it does work. It looks like justify just doesn't like getting string variables. 🤔

Reply:
nvm fixed it 👍